SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialStrategySystem.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include "Schema/CrossServerEndpoint.h"
+#include "SpatialView/SubView.h"
+#include "Utils/CrossServerUtils.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialStrategySystem, Log, All)
+
+class SpatialOSWorkerInterface;
+
+namespace SpatialGDK
+{
+class SpatialStrategySystem
+{
+public:
+	SpatialStrategySystem(const FSubView& InSubView, Worker_EntityId InStrategyWorkerSystemEntityId)
+		: SubView(InSubView)
+		, StrategyWorkerSystemEntityId(InStrategyWorkerSystemEntityId)
+	{
+	}
+
+	void Init(SpatialOSWorkerInterface* Connection);
+	void Advance(SpatialOSWorkerInterface* Connection);
+	void Flush(SpatialOSWorkerInterface* Connection);
+	void Destroy(SpatialOSWorkerInterface* Connection);
```

Comment:
This isn't called, is it?

Reply:
Good spot. Added a call from Destroy in `SpatialNetDriver::BeginDestroy`